README.md

```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
+### Support for importing 1.10+ versions of Kubernetes
+### Automic conversion of different versions of Kube resources and support for multiple version of resources
+* Even if you import different version of Kube, we can still use the same resource version to retrieve resources
+> For example, we can use `v1`, `v1beta2`, `v1beta1` version to retrieve the Deployments resources in different clusters.
+> 
+>   Notes: In 1.10 *deployments* is `v1beta1`, in 1.24 *deployments* is `v1`
+### A single API can be used to retrieve different types of resources
+* Use [`Collection Resource`](https://clusterpedia.io/docs/concepts/collection-resource/) to retrieve different types of resources, such as `Deployment`, `DaemonSet`, `StatefulSet`.
+```bash
+$ kubectl get collectionresources
+NAME            RESOURCES
+workloads       deployments.apps,daemonsets.apps,statefulsets.apps
+kuberesources   *,*.admission.k8s.io,*.admissionregistration.k8s.io,*.apiextensions.k8s.io,*.apps,*.authentication.k8s.io,*.authorization.k8s.io,*.autoscaling,*.batch,*.certificates.k8s.io,*.coordination.k8s.io,*.discovery.k8s.io,*.events.k8s.io,*.extensions,*.flowcontrol.apiserver.k8s.io,*.imagepolicy.k8s.io,*.internal.apiserver.k8s.io,*.networking.k8s.io,*.node.k8s.io,*.policy,*.rbac.authorization.k8s.io,*.scheduling.k8s.io,*.storage.k8s.io
+```
+### Diverse resource sync policies and intelligent sync of cutom resources with AA resources.
+* [Wildcards](https://clusterpedia.io/docs/usage/sync-resources/#using-wildcards-to-sync-resources) can be used to sync all types of resources within a specified group or cluster.
+* [Support for synchronizing all custom resources](https://clusterpedia.io/docs/usage/sync-resources/#sync-all-custom-resources)
+* The type of resources to be synchronized can be automatically adjusted accoriding to CRD and AA changes
+### Unify the search entry for master clusters and multi-cluster resources
+* Based on [Aggregated API](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/), the entry portal for multi-cluster retrieval is the same as that of the the master cluster(IP:PORT)
+### Very low memory usage and weak network optimization
+* Optimize informer's cache store for low memory usage for resource synchronization
+* Automatic start/stop synchronization based on cluster health status
+### High availability
+### No dependency on specific storage components
+Clusterpedia does not care about storage components and uses storage layer to interface to specific storage components,
```

Comment:
Clusterpedia does not care about storage components and uses the storage layer to attach specific storage components,